gap/digraph.gi

```diff
@@ -429,6 +444,98 @@ DigraphCons);
 InstallMethod(Digraph, "for a list, list, and list", [IsList, IsList, IsList],
 {dom, src, ran} -> DigraphCons(IsImmutableDigraph, dom, src, ran));
 
+InstallMethod(Digraph, "for a string naming a digraph", [IsString],
+function(name)
+  # edge case to avoid interfering with Digraph([])
+  if name = "" then
+    TryNextMethod();
+  fi;
+
+  # standardise string format to search database
+  name := LowercaseString(name);
+  RemoveCharacters(name, " \n\t\r");
+
+  # load database if not already done
+  DIGRAPHS_LoadNamedDiSparse6Strings();
+
+  if not name in RecNames(DIGRAPHS_NamedDiSparse6Strings) then
+    ErrorNoReturn("Named digraph not found. Please check argument 'name',\n",
+                  "or view list of available digraphs with names containing\n",
+                  "string s using ListNamedDigraphs(s).");
```

Comment:
This is a bit more standard for error messages in the Digraphs package:
```suggestion
    ErrorNoReturn("named digraph <name> not found; see ListNamedDigraphs,");
```
In particular, using `<` and `>` around the variable name in question; starting with a lowercase letter; ending with a comma; and not giving quite so much information. Since `ListNamedDigraphs` is already well documented, I think it's unnecessary to do more than just mention it.